skyportal/handlers/api/thumbnail.py

```diff
@@ -6,7 +6,7 @@
 from PIL import Image
 from baselayer.app.access import permissions, auth_or_token
 from ..base import BaseHandler
-from ...models import DBSession, Photometry, Source, Thumbnail
+from ...models import DBSession, Candidate, Photometry, Source, Thumbnail
```

Comment:
The changes to this file are due to the fact that the parent may now either be a comment or a candidate, so we add checks here